src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementService.kt

```diff
@@ -1,28 +1,10 @@
 package uk.gov.justice.digital.hmpps.welcometoprison.model
 
-import com.github.javafaker.Faker
 import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.welcometoprison.model.basm.BasmService
 import java.time.LocalDate
-import java.time.ZoneId
-import java.util.Random
 
 @Service
-class MovementService(val faker: Faker = Faker()) {
-  private val numbers = '0'..'9'
-  private val letters = 'A'..'Z'
-
-  fun CharRange.get(n: Int) = this.shuffled().take(n).joinToString("")
-
-  fun getMovements(agencyId: String, date: LocalDate) = generateSequence {
-    Movement(
-      firstName = faker.name().firstName(),
-      lastName = faker.name().lastName(),
-      dateOfBirth = faker.date().birthday().toInstant().atZone(ZoneId.systemDefault()).toLocalDate(),
-      fromLocation = faker.address().city(),
-      prisonNumber = if (Random().nextBoolean()) letters.get(1) + letters.get(4) + numbers.get(2) else null,
-      pncNumber = numbers.get(4) + "/" + numbers.get(7) + letters.get(1),
-      date = date,
-      moveType = MoveType.values().random(),
-    )
-  }.take((5..20).random()).toList()
+class MovementService(val basmService: BasmService) {
+  fun getMovements(agencyId: String, date: LocalDate) = basmService.getMoves(agencyId, date, date)
```

Comment:
this will presumably be enhanced to include prison transfers from the prison api and enriched with match data, etc..